src/App/App.js

```diff
@@ -16,35 +16,41 @@ class App extends Component {
 
   componentDidMount() {
     this.setState({
-      loading: true
+      loading: true,
     });
     getAllMovies()
       .then((data) => {
         this.setState({
           movies: data.movies,
-          loading: false
+          loading: false,
         });
       })
       .catch((err) => this.setState({ error: true }));
   }
 
   render() {
     if (this.state.error) {
-      return <ErrorPage />
+      return <ErrorPage />;
     } else if (this.state.loading) {
-      return <Loading />
+      return <Loading />;
     }
     return (
       <>
         <nav className="navbar">
-          <h1>Rancid Tomatillos</h1>
+          <Link to="/" className="link-style">
```

Comment:
Lovely. There'll still be a return to main link on the error page but this will ensure a user can always get back to the main view.